### ISSUE TITLE
MultiMC5-Cracked

### ISSUE THREAD

**Issue by Denirvan** [AfoninZ/MultiMC5-Cracked#30]
Здравствуйте! Я использую MultiMC5-Cracked с автономным профилем, так как у меня нет лицензии Minecraft я играю в оффлайн режиме. Я правильно понимаю что MultiMC5-Cracked нельзя обновлять, так как после обновления MultiMC5-Cracked требует ввод лицензионного аккаунта?

**Comment by Dakkaron** [comment]
Can you please repeat in English?

**Comment by Denirvan** [comment]
> 
> 
> Не могли бы вы повторить на английском?

Yes, of course.

**Comment by Dakkaron** [comment]
Yes, this is a very light fork from the original. The update server hasn't been changed, so if you update from the program itself, you will get the original (non-cracked) MultiMC.

If you want to update, check the download link in this repo. Since there are only manual builds, it's usually a few updates behind.

**Comment by AfoninZ** [comment]
Yep, I think updates being fetched from the official server is a bug and should be fixed - will close the issue when done so.

**Comment by Sebastian-byte** [comment]
I think updates are checked by using the build number, we could disable the build number as a workaround.

**Comment by Sebastian-byte** [comment]
Well I think that https://github.com/AfoninZ/MultiMC5-Cracked/commit/726cce8cf609a8336b527ae1e77018f4fe02f955 workarounds this issue by getting the build number from the root CI, however we should anyway implement a way of updating from the launcher.

**Comment by Vulcanoz** [comment]
So all the same, the ability to update the cracked MultiMC inside the program will be implemented (without having to reinstall the launcher)?

**Comment by Sebastian-byte** [comment]
> So all the same, the ability to update the cracked MultiMC inside the program will be implemented (without having to reinstall the launcher)?

Probably

**Comment by Samisafool** [comment]
Maybe should consider disabling update message?

**Comment by Sebastian-byte** [comment]
I don't know

**Comment by Slddev** [comment]
An auto-updater for UltimMC would be nice, but I bet it will be quite hard to implement it. (If I'm correct)

**Comment by Samisafool** [comment]
How about github pages? Would work if server is only static

**Comment by maximmasterr** [comment]
We can use vercel for serverless api that checks out github for updates, and maybe i will do that someday

**Comment by sinanisler** [comment]
please fix this @AfoninZ 
people making me crazy on my website.

its like %80 of them making this mistake and it is really anoying.

**Comment by Sebastian-byte** [comment]
Well, the "bug" of updating with upstream isn't there anymore but still would be good to have an update system for UltimMC.
